tools/bootstrap.php

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env php
+# This script verifies a development installation of LORIS by ensuring that
+# the system has all of the required dependencies such as the correct PHP and
```

Comment:
What if PHP isn't installed?

Reply:
I'll add a check for it

Reply:
I mean the script is written in PHP, so how can it check that?